Assets/ScriptableObjects/Items/KnockbackExtension.asset

```diff
@@ -15,7 +15,7 @@ MonoBehaviour:
   id: Knockback
   displayName: Knockback
   secretName: punchy
-  displayDescription: Players recieve massive knockback
+  displayDescription: Opponents and you receive massive knockback
```

Comment:
Maybe just write "everyone recieves" or alternatively if you want to underline that it happens to both parties:" **Both** you and your opponents receive massive knockback"

Reply:
Did the lengthy approach with opponent in singular for clarity.